internal/authflow/flow.go

```diff
@@ -28,37 +27,7 @@ var (
 	jsonTypeRE = regexp.MustCompile(`[/+]json($|;)`)
 )
 
-type iconfig interface {
-	Get(string, string) (string, error)
-	Set(string, string, string)
-	Write() error
-}
-
-func AuthFlowWithConfig(cfg iconfig, IO *iostreams.IOStreams, hostname, notice string, additionalScopes []string, isInteractive bool) (string, error) {
-	// TODO this probably shouldn't live in this package. It should probably be in a new package that
-	// depends on both iostreams and config.
-
-	// FIXME: this duplicates `factory.browserLauncher()`
```

Comment:
Addressed both of these as part of the refactor.

Reply:
> Addressed both of these as part of the refactor by having the `AuthFlow` take a `browser.Browser` as an input and moving the writing of the config out of this flow and having the caller be responsible for writing it.